ui/ozone/platform/wayland/wayland_window.cc

```diff
@@ -109,11 +109,22 @@ bool WaylandWindow::Initialize() {
   ui::PlatformWindowType ui_window_type =
       ui::PlatformWindowType::PLATFORM_WINDOW_TYPE_WINDOW;
   delegate_->GetWindowType(&ui_window_type);
-  if (ui_window_type == ui::PlatformWindowType::PLATFORM_WINDOW_TYPE_WINDOW) {
-    CreateXdgSurface();
-  } else {
-    CreateXdgPopup();
+  switch (ui_window_type) {
+    case ui::PlatformWindowType::PLATFORM_WINDOW_TYPE_MENU:
+    case ui::PlatformWindowType::PLATFORM_WINDOW_TYPE_POPUP:
+      // TODO(msisov, tonikitoo): Handle notification windows, which are marked
+      // as popup windows as well. Those are the windows that do not have
+      // parents and popup when a browser receive a notification.
+      CreateXdgPopup();
+      break;
+    case ui::PlatformWindowType::PLATFORM_WINDOW_TYPE_WINDOW:
+      CreateXdgSurface();
+      break;
+    default:
+      NOTREACHED() << "Not supported window type: type=" << ui_window_type;
```

Comment:
This makes the browser crashing immediately, when it tries to create tooltip windows, for example. 

We had this NOTREACHED() when tooltips were disabled on the aura level. 

Can you please make another [TEMP] patch, which unites the
case ui::PlatformWindowType::PLATFORM_WINDOW_TYPE_WINDOW:
and default cases and calls CreateXdgSurface instead?

`case ui::PlatformWindowType::PLATFORM_WINDOW_TYPE_WINDOW:`
 `default:`
`   CreateXdgSurface();`
 `  break;`

Reply:
Sorry, I realized that crash just before. I updated branch with removing converting window type at  GetWindowType() minutes ago because there is no tooltip handling yet. WDYT?

Reply:
add todo there.